app/pages/_app.tsx

```diff
@@ -135,12 +136,17 @@ export default function App({ Component, pageProps }: AppProps) {
 
 function RootErrorFallback({ error, resetErrorBoundary }: ErrorFallbackProps) {
   if (error instanceof AuthenticationError) {
-    return <LoginForm onSuccess={resetErrorBoundary} />
+    return (
+      <ErrorComponent
+        statusCode={error.statusCode}
+        title={"Sorry, please login or sign up to see this."}
```

Comment:
Please add the message, that if people are logged in they need to reload the page because sometimes authentication errors occur after loggin in.